data/initrd/scripts/udev_setup

```diff
@@ -26,6 +26,12 @@ if [ -n "$linuxrc_no_auto_assembly" ] ; then
   echo 'ENV{ANACONDA}="yes"' > /run/udev/rules.d/00-inhibit.rules
 fi
 
+# create NVMe config files before udevd is started (bsc#1184908)
+if [ -x /usr/sbin/nvme ] ; then
```

Comment:
While at it, could we also test if the hostnqn file already exists? If it does, just skip this step? 
This would allow to inject a hostnqn via the command line, e.g. 

`  echo nqn.2014-08.org.nvmexpress:uuid:1a9e23dd-466e-45ca-9f43-a29aaf47cb21 > /etc/nvme/hostnqn
`

Reply:
Command line is actually parsed later. For this to work properly we'd really need a separate linuxrc option.

Reply:
Understood. I've modified the line to 

`if [ ! -f /etc/nvme/hostnqn ] && [ -x /usr/sbin/nvme ] ; then`

and added the hostnqn file to initrd via mksusecd. My test setup with qemu doesn't have
have proper interfaces to generate the correct system uuid. Anyway with this I was
able to get it working.